src/utils/getAssetsUrl.ts

```diff
@@ -1,9 +1,17 @@
+import type { UploadedImage } from '@prezly/sdk';
+
 import { ASSETS_URL } from './constants';
 
 /**
- * @param uuid UUID of the assets hosted on Uploadcare
+ * @param image The `UploadedImage` object or uuid of the asset hosted on UploadCare
  * @returns Full asset URL that can be used as a source for <img>, <video> and <a> tags.
  */
-export function getAssetsUrl(uuid: string) {
-    return `${ASSETS_URL}/${uuid}/`;
+export function getAssetsUrl(image: UploadedImage | string) {
+    if (typeof image === 'string') {
+        return `${ASSETS_URL}/${image}/`;
+    }
+    if (image.effects.length > 0) {
+        return `${ASSETS_URL}/${image.uuid}/-${image.effects.join('-')}`;
+    }
```

Comment:
I wish we had this functionality implemented in a single place and reused everywhere.

There is `prezly/uploadcare` package that has it: https://github.com/prezly/uploadcare/

But I guess there was a reason to not use it. Is this to reduce the bundle size @oleg-semyonov ?

Reply:
Yes, that was purely bundle size decision. But I agree that it's better to have everything in one place. Maybe if the bundle size impact is not that big, we can include that it in the deps.

Reply:
@MohammadxAli can you check the impact on the bundle size? It would make more sense to use the `uploadcare` package as Ivan suggested.